pkg/daemon/ceph/client/mirror.go

```diff
@@ -280,17 +280,17 @@ func removeSnapshotSchedule(context *clusterd.Context, clusterInfo *ClusterInfo,
 	return nil
 }
 
-func EnableSnapshotSchedules(context *clusterd.Context, clusterInfo *ClusterInfo, pool string, snapshotSchedules []cephv1.SnapshotScheduleSpec) error {
+func EnableSnapshotSchedules(context *clusterd.Context, clusterInfo *ClusterInfo, poolRadosNamespaceName string, snapshotSchedules []cephv1.SnapshotScheduleSpec) error {
```

Comment:
Calling methods and vars `poolRadosNamespace` is a bit long and tedious. How about a more general var name like `resourceName`?
```suggestion
func EnableSnapshotSchedules(context *clusterd.Context, clusterInfo *ClusterInfo, resourceName string, snapshotSchedules []cephv1.SnapshotScheduleSpec) error {
```

Reply:
Actually, `poolRadosNamespaceName` gives a idea that `pool` or `pool/radosnamespace` can be passed,
We have followed this for a lot of places, either we should just change for function parameters, or look for a better name for all over the code

Reply:
Or how about just keeping it as `poolName`?

Reply:
I'm fine with either `resourceName` or `poolName`. Like in my previous comment, it just isn't necessary to include `poolRadosNamespace` everywhere. As an API, it keeps things simple and I think also intuitive since rados namespaces are inside pools.